gatsby-node/constants.js

```diff
@@ -1,7 +1,7 @@
 const {map} = require('ramda');
 const {join, resolve} = require('path');
 
-const lifeCycleStages = ['createPages', 'onCreateNode'];
+const lifeCycleStages = ['onCreateNode', 'createPages'];
```

Comment:
Why did these get flipped?

Reply:
Wanted them to be in execution order––`onCreateNode` occurs before `createPages`

Reply:
Got it :)